Serum-390/godzilla
# UI: Inventory view prototype

# Description

### Changes:

- Adds a demo inventory view.
- Adds a demo REST api to return some inventory items
- Some formatting of the JSX in NavBar

## Link to work item

Closes #28

## How to Test

1. Run the app.
2. Check the `/Inventory` route.
